### PR TITLE
add minecraft theme

### PR DESCRIPTION
**Pull request by Jarvis1010**
Here is a first draft.  Let me know what you think.

### CODE REVIEW

File: src/themes/minechat.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
```
Comment (BenDMyers) [inline src/themes/minechat.css:1]: This font is a pretty good substitute! If we wanted to be a little more authentic, it turns out there are [some pens out there](https://codepen.io/Daltontastic/pen/YWVadG) that feature the Minecraft font as a `data:font` URL. I think it's up to you if you'd like to use that.

File: src/themes/minechat.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+[data-twitch-chat] > *:last-child {
+  margin-bottom: 2rem;
+}
+
+[data-twitch-chat] [data-twitch-message] [data-twitch-message-reply] {
+  display: none;
+}
+
+/* css to add "operator crown" for the broadcaster */
+[data-twitch-sender-roles="broadcaster"] > .twitch-chat-sender::before {
```
Comment (BenDMyers) [inline src/themes/minechat.css:40]: I think broadcasters can technically have other roles as well. To be safe, we should use the `~=` operator here to check that `broadcaster` is a substring.

File: src/themes/minechat.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  --size: 1.5rem;
+  content: "";
+  display: inline-block;
+  background-color: var(--twitch-sender-color, var(--twitch-purple));
+  height: var(--size);
+  width: var(--size);
+  margin-inline-end: 1rem;
+  clip-path: polygon(50% 0%, 76% 26%, 100% 0, 100% 95%, 0 95%, 0 0, 24% 24%);
```
Comment (BenDMyers) [inline src/themes/minechat.css:48]: I absolutely love that you've added a broadcaster icon like this! 🎉 👑

File: src/themes/minechat.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+[data-twitch-sender-roles="broadcaster"] > .twitch-chat-sender::after {
```
Comment (BenDMyers) [inline src/themes/minechat.css:51]: Same as above - it'd be safest to use `~=` when checking that the sender is a broadcaster.

**Review by BenDMyers — CHANGES_REQUESTED** [review]

File: src/themes/minechat.css (hunk 1 above)
Comment (Jarvis1010) [reply src/themes/minechat.css:1]: I like that  font better, I'm updating that now

File: src/themes/minechat.css (hunk 3 above)
Comment (Jarvis1010) [reply src/themes/minechat.css:48]: Gave me a reason to use clip-path.  Its not often I find one

File: src/themes/minechat.css (hunk 2 above)
Comment (Jarvis1010) [reply src/themes/minechat.css:40]: Good catch.  I am fixing that now

File: src/themes/minechat.css (hunk 4 above)
Comment (Jarvis1010) [reply src/themes/minechat.css:51]: This is a copy paste error, I should have been to `data-twitch-message`

**Review by BenDMyers — APPROVED** [review]

**Pull request merged**